app/src/test/java/chess/PawnTest.java

```diff
@@ -0,0 +1,22 @@
+package chess;
+
+import org.junit.jupiter.api.*;
+import static org.assertj.core.api.Assertions.*;
+
+public class PawnTest {
+
+    @Test
+    @DisplayName("폰이 정상적으로 생성되는지 확인한다.")
+    public void create() {
+        final String colorWhite = "white";
+        final String colorBlack = "black";
+        verifyPawnColor(colorWhite);
```

Comment:
```suggestion
        verifyPawnColor(WHITE);
```
일반 문장처럼 코드에서도 단어의 중복은 피하고 간결하게 작성하는 게 좋습니다.

Reply:
이후 작성에서도 중복되는 의미의 반복은 피하도록 고려하겠습니다